backend/node_app/modules/base/exportHandler.js

```diff
@@ -17,11 +17,12 @@ class ExportHandler {
 		this.appSettings = APP_SETTINGS
 	}
 
-	async export(res, searchText, format, options, cloneName, permissions, userId) {
+	async export(res, searchText, classificationMarking, format, options, cloneName, permissions, userId) {
```

Comment:
move classificationMarking into options here as well. This will get expanded into body below.